packages/confirm-dialog/src/vaadin-confirm-dialog.d.ts

```diff
@@ -73,6 +73,19 @@ export type ConfirmDialogEventMap = ConfirmDialogCustomEventMap & HTMLElementEve
  * @fires {CustomEvent} opened-changed - Fired when the `opened` property changes.
  */
 declare class ConfirmDialog extends ElementMixin(ThemePropertyMixin(ControllerMixin(HTMLElement))) {
+  /**
+   * Sets the `aria-describedby` attribute of the overlay element.
+   *
+   * By default, all elements inside the message area are linked
+   * through the `aria-describedby` attribute. However, there are
+   * cases where this can confuse screen reader users (e.g. the dialog
+   * may present a password confirmation form). For these cases,
+   * it's better to associate only the elements that will help describe
+   * the confirmation dialog through this API.
+   * @attr {string} accessible-description-ref
+   */
+  accessibleDescriptionRef: string | null | undefined;
```

Comment:
I was wondering if we should restrict the type to `string`. However, I discovered that we already have properties like `errorMessage` that allow both `null` and `undefined`, so let's not change it for now then.